beluga/docs/README.md

```diff
@@ -1,12 +1 @@
 This library provides implementations for Monte Carlo based localization algorithms.
-The existing implementations are:
-
-- MCL: Monte Carlo Localization using a fixed number of particles.
-- AMCL: Adaptive Monte Carlo Localization, the number of particles is defined using the KLD criteria.
-
-The library is extensible, allowing to:
-- Provide different [sensor](@ref SensorModelPage) or [motion](@ref MotionModelPage) models.
-- Provide a way of getting the [estimated pose](@ref StateEstimatorPage) based on the particles.
-- Provide a way of [generating the initial particles](@ref StateGeneratorPage).
-- Provide a way of [sampling](@ref SamplerPage) from the previous particle set.
-- Provide a way to [limiting](@ref LimiterPage) the amount of particles.
```

Comment:
@nahueespinosa why remove all hyperlinks to library concepts? Some no longer apply, but others do (like the ones for sensor and motion models).

Reply:
I didn't see a quick way of keeping the sensor and motion model hyperlinks and removing everything else while maintaining the sense of the text. I'm working on a bigger update to the API documentation page that I'll be pushing to this branch soon.

Reply:
Done.